fuzzers/qemu_launcher/src/fuzzer.rs

```diff
@@ -174,7 +366,7 @@ pub fn fuzz() {
         .expect("Failed to create QemuExecutor");
 
         // Wrap the executor to keep track of the timeout
-        let mut executor = TimeoutExecutor::new(executor, timeout);
+        let mut executor = TimeoutExecutor::new(executor, Duration::from_secs(options.timeout));
```

Comment:
The timeout should be directly parsed in the clap options. See https://github.com/AFLplusplus/LibAFL/blob/71aa0221a01482097715e695de0fe6197dc90aed/libafl/src/bolts/cli.rs#L117

(an maybe we could even use `cli`?

Reply:
Sound good. What do you mean by cli?

Reply:
LibAFL has a `FuzzerOptions` parser in the `cli` feature (the one I linked, actually).
I'm not sure if it has all the features you need but it might be an idea, see https://github.com/AFLplusplus/LibAFL/blob/71aa0221a01482097715e695de0fe6197dc90aed/fuzzers/frida_gdiplus/src/fuzzer.rs#L60 for usage

Reply:
Fixed. I stuck with clap though as I felt it kept things easier keeping all the command line options in one place.